app/views/merchants/index.html.erb

```diff
@@ -10,7 +10,7 @@
         <% end %></h3>
       <ul>
         <% if admin_user? %>
-          <li><%= link_to 'Disable' %></li>
+          <li><%= link_to 'Disable', admin_merchant_path(merchant, disable: true), method: :patch %></li>
```

Comment:
Is this path confusing for anyone else? Would it simplify things to just hand-roll a "disable_merchant_path"

Reply:
@milevy1 Thats how I was doing it in my first attempt at the story, but me and Blake decided it was unrestful, we should decided as a group